content/articles/entra-identity-provider.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Entra as an Identity Provider
+excerpt: How to use Entra to create and log in to accounts.
+meta: Learn how to seamlessly integrate Entra as your identity provider for account creation and secure logins, enhancing your user experience and security.
+categories:
+- Account
+---
+
+# Entra as an Identity Provider
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Using Entra as an identity provider for single sign-on streamlines the login experience for you and your team and helps you manage risk at scale.
+
+<info>
+Entra as an Identity Provider is only available on Enterprise plans. See our [pricing page](https://dnsimple.com/pricing) for more details.
+</info>
+
+## Prerequisites
+
+To proceed with configuring login with SSO through Entra, you must:
+
+- Have administrator access to a DNSimple account.
+- Be subscribed to a DNSimple Enterprise plan.
+
+## Linking an Entra organization to a DNSimple account
+
+DNSimple's Entra app integration is pending review and not yet available for installation via the [Microsoft Entra App Gallery](https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-integrated-apps).
+
+### Creating a customer Entra app
+
+For now, you can use Entra as an Identity Provider by [creating your own custom <label>Single tenant</label> Entra app integration](https://docs.azure.cn/en-us/data-explorer/kusto/access-control/provision-entra-id-app?view=microsoft-fabric&tabs=portal) to work with DNSimple by following these steps. Don't forget to take note of your <label>Directory (tenant) ID</label>, <label>Application (client) ID</label>, and <label>Client secret</label>.
```

Comment:
I noticed our `label`s don't have any styling. I'd suggest removing them, but we have a lot of them across all articles.

Reply:
I've been updating these to ** ** when I come across them, hopefully will catch them all with this revamp.